app/backend/templates/v2/generated_html/account_deletion_complete.html

```diff
@@ -96,7 +96,7 @@
 
 <body style="word-spacing:normal;background-color:#eeeeee;">
   <div style="display:none;font-size:1px;color:#ffffff;line-height:1px;max-height:0px;max-width:0px;opacity:0;overflow:hidden;">{{ preview }}</div>
-  <div style="background-color:#eeeeee;" lang="und" dir="auto">
+  <div aria-label="{{ subject }}" aria-roledescription="email" style="background-color:#eeeeee;" role="article" lang="und" dir="auto">
```

Comment:
FYI I followed the readme to generate the mjml etc here:

app/backend/templates/v2/readme.md

It put all this stuff in here that's not just the activeness probe so tell me if that's not the right way anymore!